install/installer/pkg/components/spicedb/data/schema.yaml

```diff
@@ -23,6 +23,10 @@ schema: |-
     // Any user in the organization can try to leave their organization
     permission leave = owner + member
 
+    // Owners can delete the organization,
+    // and through this process also disconnect all users from the organization
+    permission delete = owner
```

Comment:
This is not the case. Only members of the installation (i.e. non org users) that are also an owner of the org can delete orgs.

Reply:
Yes. The SSO is handled as it is today, with a special case. Otherwise, it's correct that only owner can delete the org.

Once we've modelled the SSO case, we can extend here.